src/vs/platform/actions/browser/menuEntryActionViewItem.ts

```diff
@@ -135,6 +135,8 @@ export function createActionViewItem(action: IAction, keybindingService: IKeybin
 
 const ids = new IdGenerator('menu-item-action-item-icon-');
 
+// {{SQL CARBON EDIT}} - This is here to use the 'ids' generator above
```

Comment:
Remove these comments since we're not changing any of this code anymore - less conflicts with the VS stuff